app/assets/tailwind/application.css

```
@@ -13,6 +13,7 @@
 @import "./google-sign-in.css";
 @import "./date-picker-dark-mode.css";
 @import "./print-report.css";
+@import "./privacy-mode.css";
```

Stylelint (17.3.0)

[error] Unexpected invalid position `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)